src/api.ts

```diff
@@ -1,9 +1,8 @@
 import { MovieData } from './types/MovieData';
-import { ResponseError } from './types/ReponseError';
 
-const API_URL = 'https://www.omdbapi.com/?apikey=your-key';
+const API_URL = 'http://www.omdbapi.com/?i=tt3896198&apikey=fe815c7';
```

Comment:
there is some problem with api request
<img width="1091" alt="image" src="https://user-images.githubusercontent.com/66749919/205430132-cfcd9579-29a7-44d3-8886-e31dc3c2cfbc.png">

Reply:
api key not working? I haven't this problem